package.json

```diff
@@ -18,14 +19,22 @@
     "clean": "rimraf packages/*/dist"
   },
   "devDependencies": {
+    "@eslint/js": "^9.37.0",
     "@types/bun": "^1.1.11",
+    "@typescript-eslint/parser": "^8.45.0",
     "esbuild": "^0.24.0",
+    "eslint": "^9.37.0",
+    "eslint-plugin-react": "^7.37.5",
+    "eslint-plugin-react-hooks": "^6.1.1",
+    "eslint-plugin-react-refresh": "^0.4.23",
     "nodemon": "^3.1.7",
     "npm-run-all": "^4.1.5",
     "prettier": "^3.4.2",
     "prettier-plugin-tailwindcss": "^0.6.14",
     "rimraf": "^6.0.1",
-    "typescript": "^5.6.3"
+    "typescript": "^5.8.3",
+    "typescript-eslint": "^8.45.0",
+    "zod": "^4.1.11"
```

Comment:
installing zod fixed some eslint's own deps misconfiguration malarkey (not in our repo)